docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,706 @@
+
+When responding to an interaction received **via webhook**, your server can simply respond to the received `POST` request. You'll want to respond with a `200` status code (if everything went well), as well as specifying a `type` and `data`, which is an object with parameters the same as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook):
+
+```py
+@app.route('/', methods=['POST'])
+def my_command():
+    if request.json["type"] == 1:        
+        return jsonify({
+            "type": 1
+        })
+    
+    else:
+        return jsonify({
+            "type": 4,
+            "data": {
+                "tts": False,
+                "content": "Congrats on sending your command!",
+                "embeds": [],
+                "allowed_mentions": []
+            }
+        })
+```
+
+If you are receiving Interactions over the gateway, you will **also need to respond via HTTP**. Responses to Interactions **are not sent as commands over the gateway**.
+
+To respond to a gateway Interaction, make a `POST` request like this. `interaction_id` is the unique id of that individual Interaction from the receieved payload. `interaction_token` is the unique token for that interaction from the received payload:
+
+```py
+url = "https://discord.com/api/v8/interactions/<interaction_id>/<interaction_token>/callback"
+
+json = {
+    "content": "Congrats on sending your command!"
+}
+r = requests.post(url, json=json)
+```
+
+> info
+> Interaction `tokens` are valid for **15 minutes** and can be used to send followup messages.
+
+## Followup Messages
+
+Sometimes, your bot will want to send followup messages to a user after responding to an interaction. Or, you may want to edit your original response. You can use the following endpoints to do so:
+
+- `PATCH /webhooks/<interaction_id>/<interaction_token>/messages/@original` to edit your initial response to an Interaction
+- `DELETE /webhooks/<interaction_id>/<interaction_token>/messages/@original` to delete your initial response to an Interaction
+- `POST /webhooks/<interaction_id>/<interaction_token>/messages` to send a new message
+- `PATCH /webhooks/<interaction_id>/<interaction_token>/messages/<message_id>` to edit a message sent with that `token`
+
+Interaction tokens are valid for **15 minutes**, meaning you can respond and update to a Slash Command within that amount of time.
+
+## Security and Authorization
+
+If you are receiving Interactions via outgoing webhook, there are some security steps you **must** take before your app is eligible to receive requests. The internet is a scary place, especially for people hosting open, unauthenticated endpoints.
+
+Every Interaction is sent with an `x-signature-ed25519` header. The header includes an encryption key and a timestamp. Using your favorite security library of choice, you **must validate this signature header each time to you receive an interaction**. If the signature fails validate, respond with a `401` error code.
```

Comment:
I think we should elaborate here on what exactly is required. eg.
> We send `x-signature-timestamp` and `x-signature-ed255191`. Concatenate the timestamp with the request body and validate that message against the signature

Reply:
The newest docs have some sample code in there now!